tests/test_pandas_utils/test_pandas_utils.py

```diff
@@ -62,8 +70,9 @@ def test_add_lagged_numpy_columns(test_X):
         _add_lagged_numpy_columns(test_X, ["test"], 1, True)
 
 
-def test_log_step(capsys, test_df):
+def test_log_step(capsys, data):
     """Base test of log_step without any arguments to the logger"""
+    test_df = pd.DataFrame(data)
```

Comment:
I haven't worked on `log_step` in this PR, so for now I'm just keeping the `test_log_step` tests to pandas